spec/serializers/api/v1/offer_company_serializer_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+context Api::V1::OfferCompanySerializer do
+  let(:company) { create :company }
+  let(:user) { create :user }
+  let(:offer)      { create(:offer, :with_items, company: company, created_by: user) }
+  let(:serializer) { Api::V1::OfferCompanySerializer.new(offer, root: 'offer').as_json }
+
+  subject { JSON.parse( serializer.to_json ) }
+
+  it 'creates json' do
+    expect(subject['offer']['id']).to eq(offer.id)
+    expect(subject['offer']['company_id']).to eq(offer.company.id)
+    expect(subject['offer']['created_by_id']).to eq(offer.created_by.id)
```

Comment:
Can we try this `json-matcher` https://relishapp.com/waterlink/rspec-json-expectations/docs/json-expectations/include-json-matcher ?

Reply:
It's not working.. I think it's version issue.

Reply:
This is a nice idea. For time-being, if it's not working, I usually keep the data as a Hash using `as_json` and then you can use standard rspec matchers.